main.py

```diff
@@ -42,7 +42,16 @@ def get_weather():
         #         dataBaseZips.append(f"{temperature}")
         # return jsonify(dataBaseZips)
         
-
+SWAGGER_URL = '/swagger'
+API_URL = '/static/swagger.json'
+swaggerui_blueprint = get_swaggerui_blueprint(
+    SWAGGER_URL,
+    API_URL,
+    config={
+        'app_name': "My Flask Application"
+    }
+)
+app.register_blueprint(swaggerui_blueprint, url_prefix=SWAGGER_URL)

```

Comment:
move this section to line 6. setup code should be placed before method functions